.github/workflows/release.yml

```
@@ -0,0 +1,102 @@
+        uses: actions/upload-artifact@v4
+        with:
+          name: m4b-merge-windows
+          path: target/release/m4b-merge
+
+  create-release:
+    needs: release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)